website/docs/r/encryption_at_rest.html.markdown

```diff
@@ -66,9 +65,8 @@ resource "mongodbatlas_encryption_at_rest" "test" {
 * `google_cloud_kms` - (Required) Specifies GCP KMS configuration details and whether Encryption at Rest is enabled for an Atlas project.
 
 ### aws_kms
+Refer to the example in the [official github repository](https://github.com/nikhil-mongo/terraform-provider-mongodbatlas/tree/master/examples) to implement Encryption at Rest
```

Comment:
The link should be to this repo, not yours.  Just change this and I"ll get it approved.  Thanks!!